app/models/user_book.rb

```diff
@@ -5,4 +5,7 @@ class UserBook < ApplicationRecord
   validates :user_id, presence: true
   validates :book_id, presence: true
   validates :rating, presence: true, inclusion: 0..5
+
+  scope :to_read, ->(user) {where(finish_date: nil, user_id: user)}
+  scope :finished, ->(user) { where.not(finish_date: nil, user_id: user)}
```

Comment:
```suggestion
  scope :to_read, ->(user) {where(finish_date: nil)}
  scope :finished, ->(user) { where.not(finish_date: nil)}
```


So if you want to query all the user books of a certain user, you can do it like:
`user.user_books.to_read`